src/cmd/changelog.rs

```diff
@@ -228,14 +228,22 @@ impl<'a> ChangeLogTransformer<'a> {
             .map(|(title, notes)| NoteGroup { title, notes })
             .collect();
 
+        let current_tag = if todo!() {
```

Comment:
this `todo!()` probably needs to  be replaced with:

```suggestion
        let current_tag = if from_rev.0 == "HEAD" {
```

Reply:
Yeah, this is what my question was for - is there any use case where it should really link to `..HEAD`? Or shall I always turn it into the commit hash?

Reply:
I have committed your proposal, I think this is the best approach forward. I just wanted to make sure that changing the behavior is fine for you as well.